tests/test_views_search.py

```diff
@@ -314,8 +314,12 @@ def test_from_parameter_invalid_pagination(
         data = get_json(res)
         assert res.status_code == 400
         assert data['message'] == 'Invalid pagination parameters.'
-        assert {(e['field'], e['message']) for e in data['errors']} == \
-            {('from', 'Must be at least 1.'), }
+        assert \
+            {(e['field'], e['message']) for e in data['errors']} == \
```

Comment:
To avoid duplication, store the value somewhere or use a `... in (<list-of-options>)`